frontend/src/routes/__root.tsx

```diff
@@ -0,0 +1,17 @@
+import { createRootRouteWithContext, Outlet } from "@tanstack/react-router";
+import { QueryClient } from "@tanstack/react-query";
+import { Main } from "@/components/Home/Main";
+
+const rootComponent = process.env.NODE_ENV === "local" ? Main : Outlet;
+
+export const Route = createRootRouteWithContext<{
+  queryClient: QueryClient;
+}>()({
+  component: rootComponent,
+  notFoundComponent: () => (
+    <div className="flex flex-col items-center mt-8" id="error-page">
+      <h1>Oops!</h1>
```

Comment:
This is currently assigned to someone actually, make sure you close that issue so that they don't waste tiem doing this. Or re-assign to yourself and mention in a comment